### ISSUE TITLE
Sorting should be case-insensitive

### ISSUE THREAD

**Issue by samuelmr** [openwallet-foundation/digital-wallet-and-agent-overviews-sig#52]
By default, Angular sorts table rows in a case-sensitive manner. Should be case sensitive, so that for example esatus Wallet can be found at "E".

Perhaps something like this in [wallets-list.component.ts](../tree/main/viewer/src/app/wallets-list/wallets-list.component.ts#L84)
```ts
  ngOnInit() {
    this.dataSource.sortingDataAccessor = (data, sortHeaderId) => data[sortHeaderId].toLocaleLowerCase();
  }
```
Doesn't work as is due to TS typing...

**Comment by cre8** [comment]
I agree that the user wants to have `e` and `E` packed together instead `[A-Za-z]`. Feel free to open a PR for this "bug".

**Comment by samuelmr** [comment]
Fixed in #55
